### ISSUE TITLE
change default salt-master user to non-root

### ISSUE THREAD

**Issue by tlex** [sodotai/salt-root#4]
* ensure the user is created
* change the salt-master config

**Comment by tlex** [comment]
```
local:
  Name: /etc/salt/master.d/so.ai.conf - Function: file.managed - Result: Clean
  Name: /etc/salt - Function: file.directory - Result: Clean
  Name: /var/cache/salt - Function: file.directory - Result: Clean
  Name: /var/run/salt - Function: file.directory - Result: Clean
  Name: /var/log/salt - Function: file.directory - Result: Clean
  Name: salt-master - Function: service.running - Result: Clean
```

OK for merging

**Comment by tlex** [comment]
Closed in 7cda2ca1c3fb93dbd16652a33eef7b02f94e0eda
